package.json

```diff
@@ -6,32 +6,42 @@
   "scripts": {
     "predev": "yarn run minify-svg",
     "dev": "parcel src/index.html",
-    "fmt": "ocamlformat --inplace src/*.ml src/*.mli",
-    "//": [
-      "Build twice as a terrible hack because pwa-manifest runs *after* sw-cache,",
-      "so the latter doesn't know about the generated icon/webmanifest files..."
```

Comment:
I wish there was a way to add real comments to package.json...

Reply:
I thought prettier couldn't format it, but the editor warned me about a different problem.